package.json

```diff
@@ -22,6 +22,6 @@
     "typescript": "^4.0.3"
   },
   "dependencies": {
-    "obsidian-daily-notes-interface": "^0.1.0"
+    "obsidian-daily-notes-interface": "0.7.0"
```

Comment:
I upgraded this to 0.7.0 which should add support for some more complex daily notes formats, as well as add compatibility with Periodic Notes (https://github.com/liamcain/obsidian-periodic-notes)